Issue 26

Jasper-Mui opened the pull request:
### What does it fix?

Closes #26 

Add .editorconfig for c#
.editorconfig was generated using Visual Studio 2019.

I have added two properties though they may not work depending on the plugin which is noted in the EditorConfig documentation. Otherwise, the generate seems to adhere to your coding styles.

spaces_around_operators = true
curly_bracket_next_line = true

I referenced https://editorconfig.org/ which is the official documentation that Microsoft also mentions.

### How has it been tested?

Tried changing the code's format and see if it would auto-format back. 

<br/>
Thanks for considering this.

idormenco commented:
@Jasper-Mui thanks a lot for contributing. Looks good !

idormenco commented:
@Jasper-Mui if it is not too much to ask ,can you please apply editorconfig code style for all file in solution ?

Jasper-Mui commented:
I have applied the editorconfig code style to all the files in the solution. I ended up doing it manually as I could not find a tool that would automate this with the visual studio editorconfig plugin. Only 4 files ended up with styling change with spacing and a newline. Please review the changes as I am not an expert at C#

Two changes I made to the editorconfig. Moved editorconfig in the src directory as that is where the .sln file is where it was prior in the root of the git repo. So when the project is opened visual studio will pick up the .editconfig properly. 

Secondly, I removed the following as the visual studio editorconfig does not actually parse these properties and there are similar properties being applied. These were the properties I added myself previously.

spaces_around_operators = true
curly_bracket_next_line = true
